test/testdata/resolver/aliased_mixin_ordering.rb

```diff
@@ -0,0 +1,46 @@
+# typed: strict
+
+module Stringy
+  extend T::Sig
+
+  sig {returns(String)}
+  def f
+    "1"
+  end
+end
+
+module Inty
+  extend T::Sig
+
+  sig {returns(Integer)}
+  def f
+    1
+  end
+end
+
+class MixesAlias
+  extend T::Sig
+
+  StringyAlias = Stringy
+
+  include StringyAlias
+  include Inty
+end
```

Comment:
is it worth having a `MixesAliasSwapped` case?

Reply:
I think this covers it well enough